src/app/cookie/index.njk

```diff
@@ -1,159 +1,131 @@
-  </p>
-
-  <table class="govuk-table">
-    <thead class="govuk-table__head">
-      <tr class="govuk-table__row">
-        <th class="govuk-table__header" scope="col"><p class="govuk-body-s"><strong>Name</strong></p></th>
-        <th class="govuk-table__header" scope="col"><p class="govuk-body-s"><strong>Purpose</strong></p></th>
-        <th class="govuk-table__header" scope="col"><p class="govuk-body-s"><strong>Expires</strong></p></th>
-      </tr>
-    </thead>
-    <tbody class="govuk-table__body">
-      <tr class="govuk-table__row">
-        <td class="govuk-table__cell"><p class="govuk-body-s">csrftoken</p></td>
-        <td class="govuk-table__cell"><p class="govuk-body-s">Contains a string of random characters that is required to perform actions and that an attacker cannot guess</p></td>
-        <td class="govuk-table__cell"><p class="govuk-body-s">1 year</p></td>
-      </tr>
-    </tbody>
-  </table>
-
-
-  <h3 class="govuk-heading-s">Our introductory message</h2>
+    <h2 class="govuk-heading-m">{{__('heading_how_cookies_are_used')}}</h2>
 
-  <p class="govuk-body-m">
-    You may see a pop-up welcome message when you first visit Submit a GAR. We’ll store a cookie so that your computer knows you’ve seen it and knows not to show it again.
-  </p>
+    <h3 class="govuk-heading-s">{{__('subheading_how_cookies_are_used_1')}}</h2>
 
-  <table class="govuk-table">
-    <thead class="govuk-table__head">
-      <tr class="govuk-table__row">
-        <th class="govuk-table__header" scope="col"><p class="govuk-body-s"><strong>Name</strong></p></th>
-        <th class="govuk-table__header" scope="col"><p class="govuk-body-s"><strong>Purpose</strong></p></th>
-        <th class="govuk-table__header" scope="col"><p class="govuk-body-s"><strong>Expires</strong></p></th>
-      </tr>
-    </thead>
-    <tbody class="govuk-table__body">
-      <tr class="govuk-table__row">
-        <td class="govuk-table__cell"><p class="govuk-body-s">seen_cookie_message</p></td>
-        <td class="govuk-table__cell"><p class="govuk-body-s">Saves a message to let us know that you've seen our cookie message</p></td>
-        <td class="govuk-table__cell"><p class="govuk-body-s">1 month</p></td>
-      </tr>
-    </tbody>
-  </table>
+    <p class="govuk-body-m">{{__('how_cookies_are_used_1_caption_1')}}</p>
+    <p class="govuk-body-m">{{__('how_cookies_are_used_1_caption_2')}}</p>
 
-  <h3 class="govuk-heading-s">Session information</h2>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>{{__('how_cookies_are_used_1_list_item_1')}}</li>
+      <li>{{__('how_cookies_are_used_1_list_item_2')}}</li>
+      <li>{{__('how_cookies_are_used_1_list_item_3')}}</li>
+      <li>{{__('how_cookies_are_used_1_list_item_4')}}</li>
+    </ul>
 
-  <p class="govuk-body-m">
-    In order to improve usability and retain information about your account and privileges, we store a unique identifier for each session
-  </p>
+    {% macro table_header() %}
```

Comment:
Created a pair of macros to create the header (which is the same markup for all the tables) and the rows (taking in the 3 column values as parameters)